CMake/Dependencies/libcorejson-add-cmakelists.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Is this added by mistake?

Reply:
No, that's not a mistake. There is no CMakeLists.txt in coreJSON repo. And I add that file into that repo in compile time by applying this patch.